ql/src/java/org/apache/hadoop/hive/ql/exec/MoveTask.java

```diff
@@ -131,6 +134,40 @@ private boolean moveFilesUsingManifestFile(FileSystem fs, Path sourcePath, Path
     return false;
   }
 
+  public void flattenUnionSubdirectories(Path sourcePath) throws HiveException {
+    try {
+      FileSystem fs = sourcePath.getFileSystem(conf);
+      LOG.info("Checking {} for subdirectories to flatten", sourcePath);
```

Comment:
use log format 
```
      LOG.info("Checking {} for subdirectories to flatten", sourcePath);
```

Reply:
It is already on logger format, right? Did I miss something

Reply:
LOL, I guess I commented on the old PR, copied here without noticing that's already fixed